src/test/java/org/mule/extension/db/unit/MuleMySqlLoggerEnhancerFactoryTestCase.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.db.unit;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+import org.mule.extension.db.api.logger.MuleMySqlLogger;
+import org.mule.extension.db.internal.domain.logger.MuleMySqlLoggerEnhancerFactory;
+
+import static org.mockito.Mockito.verify;
+
+public class MuleMySqlLoggerEnhancerFactoryTestCase {
+
+  @Test
+  public void verifyMuleMySqlLoggerClassIsCalled() {
+    MuleMySqlLogger muleMySqlLogger = Mockito.spy(MuleMySqlLoggerEnhancerFactory.getEnhancedLogger());
```

Comment:
this is still not testing nothing as you're calling the stub directly. @nicomz can you please sync up with Julian an share our discussion? Also keep in mind that there has to be two versioons of this test, each using each driver

Reply:
done. Regarding testing each driver version: there two Maven profiles for each, so the test will be run on both.